zbus/src/blocking/proxy/mod.rs

```diff
@@ -159,7 +159,7 @@ impl<'a> Proxy<'a> {
 
     /// Introspect the associated object, and return the XML description.
     ///
-    /// See the [xml](xml/index.html) module for parsing the result.
+    /// See the [xml](../../../zbus_xml/index.html) module for parsing the result.
```

Comment:
This won't work for docs.rs. We need external link to docs.rs. The existing link is from the time when `zbus_xml` was just a mod of `zbus`.

Reply:
Sorry, I just mimicked what is done elsewhere, for example [here](https://github.com/dbus2/zbus/blob/main/zbus/src/error.rs#L28) for zvariant or zbus_names crates

What is the syntax to make the correct links ?

Reply:
Oh! I didn't know/remember docs.rs can apparently [handle this just fine](https://docs.rs/zbus/latest/zbus/enum.Error.html#variant.Variant). :open_mouth: Never mind then. :)

Reply:
ah wait, the link to zvariant **is** broken even. So that doesn't actually work.

> What is the syntax to make the correct links ?

Simply link to the https://docs.rs/zbus_xml . We need to do that same for the zvariant link.

Reply:
Ah, sorry, I didn't see that zvariant link was also broken. I'll fix these too tomorrow.
Thank you for your quick replies.